components/server/src/installation-admin/installation-admin-controller.ts

```diff
@@ -16,7 +16,7 @@ export class InstallationAdminController {
         const app = express();
 
         app.get('/data', async (req: express.Request, res: express.Response) => {
-            const data = await this.installationAdminDb.getTelemetryData();
+            const data = await this.installationAdminDb.getData();
 
             res.send(data);
```

Comment:
nit: `res.status(200).json(data);` also sets the proper content header. Feel free to ignore, as you're managing the only client here :+1: